docker/base-deps/Dockerfile

```diff
@@ -88,6 +88,22 @@ RUN sudo apt-get update -y && sudo apt-get upgrade -y \
     else sudo apt-get autoremove -y wget; \
     fi;) \
     && sudo rm -rf /var/lib/apt/lists/* \
-    && sudo apt-get clean
+    && sudo apt-get clean \
+    && (if [ "$AUTOSCALER" = "autoscaler" ]; \
```

Comment:
move autoscaler library dependencies here so we can get rid of the ray-deps layer